### PR TITLE
[13.0][MIG] web_widget_url_advanced

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
**web_widget_url_advanced** migration to v13.

cc ~ @LoisRForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM!

**Comment by simahawk** [comment]
would you mind considering https://github.com/OCA/web/pull/1234#issuecomment-511708931 to merge web_widget_prefixed_url into this?

**Comment by JoanSForgeFlow** [comment]
@simahawk Of course, it's already done. Would you mind to review?

File: web_widget_url_advanced/static/src/js/url_widget.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            // via `text_field` attribute or `options.text_field`
+            var text_field = this.attrs.text_field || this.attrs.options.text_field;
+            if (text_field) {
+                var field_value = this.recordData[text_field];
+                if (_.isObject(field_value) && _.has(field_value.data)) {
+                    field_value = field_value.data.display_name;
+                }
+                this.attrs.text = field_value;
+            } else {
+                var prefix = this.attrs.prefix_name || this.attrs.options.prefix_name;
```
Comment (simahawk) [inline web_widget_url_advanced/static/src/js/url_widget.js]: I don't think this is correct/enough.
The `prefix_name` - as the word says - should be used to add a prefix to the link, such as `mailto:`.
Please check here how it's done https://github.com/OCA/web/blob/11.0/web_widget_prefixed_url/static/src/js/widget_prefixed_url.js#L20

File: web_widget_url_advanced/templates/assets.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2018 Simone Orsi - Camptocamp SA
+    License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+-->
+<odoo>
```
Comment (simahawk) [inline web_widget_url_advanced/templates/assets.xml:6]: can you rebase on 13.0 and run pre-commit? There's a fix for bad xml formatting.

File: web_widget_url_advanced/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Extend URL widget features to ease customization of anchor's text.
+
+By default, the URL widget will show the plain URL.
+You can specify an attribute to provide a static label too.
+
+With this module, you can use a field for link's inner text using attribute
```
Comment (simahawk) [inline web_widget_url_advanced/readme/DESCRIPTION.rst:6]: this must be fixed according to `prefix_name` correct behavior.

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by JoanSForgeFlow** [comment]
Sorry, I had misunderstood the module's functionality. Now, the `prefix_name` adds a prefix to the link.
Thanks for your review.

**Review by simahawk — APPROVED** [review]

**Pull request merged**